### DISCUSSION TITLE
Sync by manually pausing screens.

### DISCUSSION THREAD

**Discussion by Plebster** [f1multiviewer/issue-tracker#73] [Feature requests] [answered]
This is brilliant!!

Is there an ability to sync screens by pausing and then resuming? Coming from F1Viewer this is how I have synced before, using the space bar or click the pause buttun, nudging forward to a landmark and then resume when another screen reaches it. 

I can do this on the replays but when I try it on live the screens seem to 'drift' back to some offset... Is this something I am doing wrong or could this be a request rather than using the curser arrows?

**Comment by f1multiviewer** [comment, marked as answer]
During replays, there's a much better way, as long as you're watching a race that was synced by someone and contributed to F1MV:

- Open the streams you want
- Seek to the moment in the race you want (e.g when the formation lap starts, or post intro)
- Press `Shift+Space` to pause all streams at once
- Press `S` to sync all streams with each other

This leads to an almost perfect sync, as you can see here: https://streamable.com/q19cyx


For live broadcasts, the reason they 'drift back to the same point', is because F1MV tries to keep the players in sync with their configured `liveLatency`. Instead of pausing them, you should tweak this number up or down until they sync (they should sync pretty well out of the box, but improvements are planned here too).

You can read more here: https://beta.f1mv.com/docs/usage/syncing-streams

**Reply by f1multiviewer** [reply]
At some point, what I envision is:

One person sacrifices themselves to sync each driver, and then uses a similar mechanism to share these syncs with others, and they can then use these offsets as-is.

For now though, the spacebar isn't supported and likely won't be during live streams
